package iegad.net;


message msg_basic {
//TCPÏûÏ¢
required int32 msg_type = 1;
required int32 msg_flag = 2;
required string msg_bdstr = 3;
};